pom.xml

```diff
@@ -342,6 +342,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.0</version>
```

Comment:
the explicit version was missing at all

Reply:
👏

Reply:
```suggestion
```
I think we can and better to delete this one because #270 can pass all tests with all jdks without enforcing this version :)